styles/index.css

```diff
@@ -20,6 +20,16 @@ html {
 body {
   @apply bg-gray-100;
 }
+
+#__next {
+  @apply flex;
+  @apply flex-col;
+  @apply min-h-screen;
+}
+
+.bg-base-secondary {
```

Comment:
Just put this here temporarily. Let's figure out where all the color vars should go.

Reply:
colors and other tokens should be in `tailwind.config.js`. custom classes in `index.css`.